ocaml/idl/datamodel_pool.ml

```diff
@@ -21,6 +21,9 @@ let operations =
       ; ( "sync_updates"
```

Comment:
I think the 2 cases are both syncing updates:
1. sync updates from remote repo
2. sync updates from local bundle repo
Then how about rename `sync_updates` to `sync_remote`?

Reply:
This will change API, so just keep it.

Reply:
Yes, I see, as the `sync_updates` operation is corresponding to the `sync_updates` api.